### PR TITLE
docs/fleerp: combine pages into a single one with multiple headers

### LINKED ISSUE #16: docs/fleerp: combine pages into a single one with multiple headers
Some pages could be combined into a single one with multiple sections. 

For example "Активиране на GPS устройство":
image

The content of SubPages "Начало", "Автоматично или Ръчно", "Проследяващи устройства" could be moved into a single page "Активиране на GPS устройство" as separate sections.

### PR DESCRIPTION
**Pull request by IvelinYR**
fixes #16

Now we combined some of Introduction pages into a single one with the first pages.

Screenshot from 2019-07-23 15-28-12

### CODE REVIEW

**Comment by mgenov** [comment]
The root menu levels are not required in this case as they are not providing so much information:

image

For example: Активиране на GPS устройство and Процес по активиране could be combined into a single point: Активиране на обекти зa проследяване (Активиране на ново у-во)

**Comment by IvelinYR** [comment]
Done.

Screenshot from 2019-07-24 09-44-37

**Comment by mgenov** [comment]
Entry page of Fleerp seems broken in the build: 

image

This happens when I try to access https://docs-pwa0mwqr9.now.sh/bg/fleerp/docs/get-started/user-account/user-registration

Keep in mind that a similar strategy should be performed for all pages.

**Comment by IvelinYR** [comment]
Done.
I had forgotten to change the root of the button `Fleerp`.
Now button redirect to correctly URL.

File: src/pages/fleerp/docs/get-started/tracker-activation.bg.md [hunk 1]
```diff
@@ -1,11 +1,54 @@
 ---
 date: "2019-07-20"
-title: Проследяващи устройства
+title: Активиране на ново устройство
```
Comment (mgenov) [inline src/pages/fleerp/docs/get-started/tracker-activation.bg.md]: Активиране на устройство

File: src/pages/fleerp/docs/get-started/tracker-activation.bg.md [hunk 2]
```diff
@@ -1,11 +1,54 @@
...
 root: "/fleerp"
-parents: ["Първи стъпки", "Активиране на GPS устройство"]
-priority: 5
+parents: ["Първи стъпки"]
+priority: 3
 ---
 
+# Активиране на устройство
```
Comment (mgenov) [inline src/pages/fleerp/docs/get-started/tracker-activation.bg.md]: Въведение

**Comment by IvelinYR** [comment]
Done.

**Review by mgenov — APPROVED** [review]
LGTM !

**Pull request merged**